delegatedaccess/impl/src/java/org/sakaiproject/delegatedaccess/logic/ProjectLogicImpl.java

```diff
@@ -234,7 +230,7 @@ public void updateNodePermissionsForUser(NodeModel nodeModel, String userId){
 			// Remove any existing notifications for this node
 	    	scheduledInvocationManager.deleteDelayedInvocation("org.sakaiproject.delegatedaccess.jobs.DelegatedAccessShoppingPeriodJob", nodeModel.getNode().id);
 			//update the shopping period site settings (realm, site properties, etc)
-			scheduledInvocationManager.createDelayedInvocation(timeService.newTime(),
+			scheduledInvocationManager.createDelayedInvocation(new Date().toInstant(),
```

Comment:
I think you can just use `Instant.now()`

Reply:
@buckett OK